<!-- person_tracker.launch -->
<launch>

  <!-- Launch the person tracker script -->
  <node name="person_tracker" pkg="mattbot_image_detection" type="person_tracker.py" output="screen">
  </node>

</launch>
